### DISCUSSION TITLE
B站已经把直播签到删了吗

### DISCUSSION THREAD

**Discussion by andywang425** [andywang425/BLTH#505] [General]
现在网页右上角的那个直播签到按钮好像没了？直播等级也没了。但是脚本的直播签到功能还能用：
![image](https://github.com/user-attachments/assets/f8cd9b9f-7e1c-4300-adb4-6ee210437dc3)

**Comment by ADJazzzz** [comment]
没删，签到能拿两个辣条礼物，这功能可以留着

**Reply by andywang425** [reply]
真的吗我咋找不到按钮了。还是说前端删了但后端API接口没删？

**Reply by ADJazzzz** [reply]
> 真的吗我咋找不到按钮了。还是说前端删了但后端API接口没删？

网页不知道，手机端在我的-我的直播里可以看见
